sycl/source/detail/builtins_relational.cpp

```diff
@@ -247,21 +247,21 @@ MAKE_1V_2V_FUNC(FOrdLessThanEqual, __vFOrdLessThanEqual, s::cl_long,
 MAKE_1V_2V_FUNC(FOrdLessThanEqual, __vFOrdLessThanEqual, s::cl_short,
                 s::cl_half, s::cl_half)
 
-// (LessOrGreater)        // islessgreater
-__SYCL_EXPORT s::cl_int LessOrGreater(s::cl_float x, s::cl_float y) __NOEXC {
-  return __sLessOrGreater(x, y);
+// (FOrdNotEqual)         // islessgreater
```

Comment:
@Nuullll, please, follow [Contribution guide](https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md#contribution-process) and pay attention to [ABI policies](https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md) in particular.
I think we can't remove `LessOrGreater` immediately and should rather deprecate them first.

Reply:
Thanks @bader. That makes sense since this is an ABI-breaking change.
Then may I ask how to deprecate an ABI?

Reply:
> how to deprecate an ABI?

@romanovvlad, @s-kanaev, please, add an answer to this question to [ABI Policy Guide](https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md).